samples/QmlArExample/qml/main.qml

```diff
@@ -30,7 +30,11 @@ ApplicationWindow {
     property Camera originCamera: null
 
     // Is true if the current scene is tabletop scene.
-    property bool tabletop: false
+    property bool tabletopMode: false
+
+    // Is true before the initial transformation is not set (tabletop mode) or
+    // before the location data source starts to received data.
+    property bool waitingInitialization: false;
```

Comment:
```suggestion
    property bool waitingForInitialization: false;
```

There may be a better name still. Maybe just `property bool initialized: false;`?